### ISSUE TITLE
Questions about usage of GT Camera

### ISSUE THREAD

**Issue by KEH0T0** [facebookresearch/vggt#252]
Hi, thank you for the great work!

According to the paper (Section "Coordinate Frame"), the predicted camera poses are defined in the coordinate frame of the first camera. Specifically, the first camera's extrinsics are set to identity (i.e., q₁ = [0, 0, 0, 1], t₁ = [0, 0, 0]). So the other frame's camera should be set to relative pose.

However, when looking at the training code, it seems that the ground-truth poses are used directly as provided by the dataset, without aligning the first camera to identity and the other to relative pose. That is, the GT poses appear to be raw world-to-camera (w2c) extrinsics. #229 

Could you please clarify:
1. Are the GT poses aligned so that the first pose becomes identity during training? Where can i find that code?
2. If not, how does the model predict first frame pose for GT poses?

Thanks in advance for your clarification!

<img width="1196" height="499" alt="Image" src="https://github.com/user-attachments/assets/18b30f50-f79c-4834-bd3a-eeb10dcd32a5" />

**Comment by jytime** [comment]
https://github.com/facebookresearch/vggt/blob/97bbde571faddde3ace3cfa7724a20448026c4c8/training/train_utils/normalization.py#L82
